src/dissect/dissect.c

```diff
@@ -61,6 +66,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "discard",   required_argument, NULL, ARG_DISCARD   },
                 { "root-hash", required_argument, NULL, ARG_ROOT_HASH },
                 { "fsck",      required_argument, NULL, ARG_FSCK      },
+                { "verity-data",   required_argument, NULL, ARG_VERITY_DATA   },
```

Comment:
please rebreak table, so that everything is nicely aligned again

Reply:
missed this one, fixed